pkg/controller/issueapi/logic.go

```diff
@@ -33,6 +33,38 @@ import (
 	"go.opencensus.io/stats"
 )
 
+var (
+	// scrubbers is a list of known twilio error messages that contain the send to phone number.
+	scrubbers = []struct {
+		prefix string
+		suffix string
+	}{
+		{
+			prefix: "phone number: ",
+			suffix: ", is not currently",
```

Comment:
Is this the only possible suffix? Should we be more gracious and just look up to the comma + space?